### PR TITLE
Format the coded

### PR DESCRIPTION
**Pull request by TriPSs**

### CODE REVIEW

**Comment by amilajack** [comment]
This breaks a lot of the code style conventions of this project. Try running `npm run lint` to see the errors. You could also try using a linter plugin.

**Comment by TriPSs** [comment]
Yeah saw It a little to late, gonna fix this tomorrow.

File: app/containers/HomePage.js [hunk 1]
```diff
@@ -8,7 +8,6 @@ import { connect } from 'react-redux';
 import * as HomeActions from '../actions/homePageActions';
 import Home from '../components/home/Home';
 
-
```
Comment (amilajack) [inline app/containers/HomePage.js]: A convention of the project is to use two new lines after imports. I have to make an eslint rule that enforces this.

**Comment by TriPSs** [comment]
@amilajack Updated the linter and got rid of most of the error, is it maybe an idea to create an PHPStorm/Webstorm code style and put it in the project? So every one who wands to work on the project and uses one of the IDEs can use it

File: package.json [hunk 2]
```diff
@@ -182,5 +182,8 @@
   "devEngines": {
     "node": "6.x",
     "npm": "3.x"
+  },
+  "dependencies": {
+    "eslint-plugin-fp": "^2.2.0"
```
Comment (amilajack) [inline package.json]: This project doesnt use "dependencies". Everything is a "devDep'

File: app/components/show/Show.js [hunk 3]
```diff
@@ -19,69 +19,49 @@ export default class Show extends Component {
...
         <ul>
           <li><h3>Season overview:</h3></li>
           <li>
             <a>
-              {seasons.length &&
-                selectedSeason &&
-                seasons[selectedSeason]
-                  ? seasons[selectedSeason].overview
-                  : null}
+              {seasons.length && selectedSeason && seasons[selectedSeason] ? seasons[selectedSeason].overview : null}
```
Comment (amilajack) [inline app/components/show/Show.js]: This previous format is more readable.

File: app/components/show/Show.js [hunk 4]
```diff
@@ -19,69 +19,49 @@ export default class Show extends Component {
...
         <ul>
           <li><h3>Episode overview:</h3></li>
           <li>
             <a>
-              {episodes.length &&
-                selectedSeason &&
-                episodes[selectedEpisode]
-                  ? episodes[selectedEpisode].overview
-                  : null}
+              {episodes.length && selectedSeason && episodes[selectedEpisode] ? episodes[selectedEpisode].overview : null}
```
Comment (amilajack) [inline app/components/show/Show.js]: This previous format is more readable.

File: app/components/show/Show.js (hunk 3 above)
Comment (TriPSs) [inline app/components/show/Show.js]: Did this because the linter was complaining that the sentence was to long.

File: package.json (hunk 2 above)
Comment (TriPSs) [inline package.json]: Did not notice this, i needed to install some things because the linter did not work.

File: app/components/movie/Movie.js [hunk 5]
```diff
@@ -548,23 +469,18 @@ export default class Movie extends Component {
...
-               <Show
-                 selectShow={this.selectShow.bind(this)}
-                 seasons={seasons}
-                 episodes={episodes}
-                 selectedSeason={selectedSeason}
-                 selectedEpisode={selectedEpisode}
-               />
-                :
-               null}
+              {activeMode === 'shows' ? <Show
```
Comment (amilajack) [inline app/components/movie/Movie.js]: This previous format is more readable.

File: package.json [hunk 6]
```diff
@@ -188,6 +188,7 @@
   },
   "optionalDependencies": {
     "chromecast-player": "0.2.3",
-    "castv2-webpack": "^0.1.10"
+    "castv2-webpack": "^0.1.10",
+    "eslint-plugin-fp": "^2.2.0"
```
Comment (amilajack) [inline package.json]: This is not an optional dep

**Comment by amilajack** [comment]
Ignore all the comments on the outdated diffs. There some formatting issues that I can't enforce with eslint. In the case of mapping over elements, try this approach instead:

```js
{items.map(item => (
  <Card
    someValue={...}
  />
))}
```

File: app/components/card/Card.js [hunk 7]
```diff
@@ -14,10 +13,7 @@ export default class Card extends Component {
     image: PropTypes.string,
     id: PropTypes.string,
     genres: PropTypes.array,
-    rating: PropTypes.oneOfType([
-      PropTypes.number,
-      PropTypes.string
-    ]),
+    rating: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
```
Comment (amilajack) [inline app/components/card/Card.js]: Please break this array into new lines to make it more readable

File: app/components/movie/Movie.js [hunk 8]
```diff
@@ -12,10 +12,7 @@ import Torrent from '../../api/Torrent';
 import CardList from '../card/CardList';
 import Rating from '../card/Rating';
 import Show from '../show/Show';
-import {
-  convertFromBuffer,
-  startServer
-} from '../../api/Subtitle';
+import { convertFromBuffer, startServer } from '../../api/Subtitle';
```
Comment (amilajack) [inline app/components/movie/Movie.js]: All multiple imports should be broken down into new lines.

**Comment by TriPSs** [comment]
Added some small fixes, saw that my IDE reseted some things.

What did you mean by the code above? If i look in the code the mapping is that way.

File: app/components/movie/Movie.js [hunk 9]
```diff
@@ -445,136 +459,118 @@ export default class Movie extends Component {
...
-                  onClick={
-                    this.startTorrent.bind(
-                      this,
-                      this.state.idealTorrent.magnet,
-                      this.state.idealTorrent.method,
-                      this.state.item
-                    )
-                  }
-                  disabled={!this.state.idealTorrent.quality}
+                  onClick={() => this.startTorrent(idealTorrent.magnet, idealTorrent.method)}
```
Comment (amilajack) [inline app/components/movie/Movie.js]: I think you forgot to pass `this.state.item` to this

**Comment by amilajack** [comment]
I've updated my comment. Lmk if it doesnt make sense. Also sorry for the inconvenience of this. There should custom ESLint rules to enforce these kinds of things.

**Comment by amilajack** [comment]
@TriPSs This will should be merged into the 0.0.6 release. 0.0.5 will be published today.

**Comment by TriPSs** [comment]
@amilajack If you agree to the linting there is in this one i can make a pull request to the other release branch. Otherwise i need to check it first again but cannot say when i will have the time to do so.

**Comment by amilajack** [comment]
Great! I will look over the code and see if I find anything.

**Comment by amilajack** [comment]
@TriPSs This has merge conflicts. Can you please fix them so I can merge?

**Comment by TriPSs** [comment]
Will create an new pull request to the new branch.

**Pull request closed without merging**